app/utils/estimate_prioritization/pooling_functions.py

```diff
@@ -17,6 +18,16 @@ def get_unique_value(series: pd.Series, default: Any = pd.NA) -> Any:
     else:
         return default
 
+def weighted_average(df: pd.DataFrame, values: str, weights: str) -> float:
```

Comment:
nit: probably clearer to name these `value_col` and `weight_col` instead

Reply:
done